bundle/regal/lsp/completion/providers/rulerefs/rulerefs_test.rego

```diff
@@ -98,11 +100,11 @@ another_local_rule := imp`])
 		},
 		"context": {"location": {
 			"row": 10,
-			"col": 21,
+			"col": 26,
```

Comment:
Not sure how this worked before 😅 I've used `|` to mark `col` 21 and 26 respectively below, and col 21 is in the middle of `:=` 🤔 

```
another_local_rule :|= imp|
```

Either way, after the change to use `location.ref_at` that function found no word at the given location, and would thus determine the "word" to be `""` — which typically means the completion was manually invoked and that all possible completions should be returned.